tests/k8s-azure/manifest/linux-vmss.json

```diff
@@ -0,0 +1,52 @@
+{
```

Comment:
Could you copy this from linux.json and only change `"availabilityProfile": "VirtualMachineScaleSets"` here

Reply:
I did the same Sir

Reply:
still not same, could you do it again and make a force push?

Reply:
Pushed